api/app/controllers/transaction.controller.ts

```diff
@@ -279,3 +281,48 @@ export const getAllTransactions = async (req: Request, res: Response) => {
     res.status(500).json({ message: "Internal server error" });
   }
 };
+
+export const processUnpaidReviewTransaction = async (
+  req: Request,
+  res: Response
+) => {
+  const reviewId = req.body.reviewId;
+  if (!reviewId) {
+    return res.status(400).send({ message: "Review id is required" });
+  }
+  try {
+    const review = await Review.findByPk(reviewId);
```

Comment:
Possibly we could use combined query here also.

Reply:
addressed here -> https://github.com/bitcointranscripts/transcription-review-backend/pull/270/commits/c43cc39fdfa3a3e68c8f47405652ce0e02ab2204